kse/src/org/kse/gui/actions/resources.properties

```diff
@@ -99,6 +99,10 @@ DeleteMultipleEntriesAction.statusbar             = Delete selected entries
 DeleteMultipleEntriesAction.text                  = Delete
 DeleteMultipleEntriesAction.tooltip               = Delete selected entries
 
+CompareCertificateAction.statusbar             = Compare certificates format ASN.1
+CompareCertificateAction.text                  = Compare
+CompareCertificateAction.tooltip               = Compare certificates
+
```

Comment:
Here is the French version, for:
- kse/src/org/kse/gui/actions/resources_fr.properties

```ini
CompareCertificateAction.statusbar             = Comparer les certificats au format ASN.1
CompareCertificateAction.text                  = Comparer
CompareCertificateAction.tooltip               = Comparer les certificats
```
_Could you update the French resource? or Could you need a PR from me?_ 
Regards.

Reply:
add a new validation and a new input is needed

CompareCertificateAction.onlytwo.message

Reply:
See new remark.